README.md

```diff
@@ -1,2 +1,45 @@
 # comp110-worksheet-B
```

Comment:
Flowchart looks OK. The "loop A" structure you have used is a little non-standard, but it's fairly clear what you mean. Similarity calculation would be clearer if `i=0` initialisation was in a separate block above the loop, and "increment correct vals" is unclear.